xdrip-get-entries.sh

```diff
@@ -8,9 +8,43 @@ main()
 
 # Cmd line args - transmitter $1 is 6 character tx serial number
   transmitter=$1
-  cmd_line_mode=${2:-""}
-  pumpUnits=${3:-"mg/dl"}
-  meterid=${4:-"000000"}
+  if [ -z  "$transmitter" ] || [ "$transmitter" == "-" ]; then
+    # check config file
+    transmitter=$(cat ${CONF_DIR}/xdripjs.json | jq -M -r '.transmitter_id')
+  fi
+  if [ -z  "$transmitter" ] || [ "$transmitter" == "null" ]; then
+    log "ERROR: No transmitter id set!; exiting"
+    exit
+  fi
+
+  cmd_line_mode=${2}
```

Comment:
Just a style thing but since we aren't automatically assigning a default at this point, maybe remove the brackets and just use $2

Reply:
It's an artifact of removing the default and should be simplified.  I'll clean it up.